2.4/root/usr/bin/run-mongod

```diff
@@ -63,12 +49,15 @@ if [ $# -gt 0 ] && [ "$1" == "initiate" ]; then
   exec ${CONTAINER_SCRIPTS_PATH}/initiate_replica.sh
 fi
 
-# Generate config file for MongoDB
-envsubst < ${CONTAINER_SCRIPTS_PATH}/mongodb.conf.template > $MONGODB_CONFIG_PATH
+# If user provides own config file use it and do not generate new one
+if [ ! -s $MONGODB_CONFIG_PATH ]; then
+  # Generate config file for MongoDB
+  envsubst < ${CONTAINER_SCRIPTS_PATH}/mongodb.conf.template > $MONGODB_CONFIG_PATH
+fi
 
 # Need to cache the container address for the cleanup
 cache_container_addr
-mongo_common_args="-f $MONGODB_CONFIG_PATH --oplogSize 64"
+mongo_common_args="-f $MONGODB_CONFIG_PATH --dbpath $MONGODB_DATADIR"
```

Comment:
Hmm, wasn't it the whole point to allow users to pass in whatever config they want? Weird to me to overwrite the dbpath.

Our default config should point to the correct dir, and perhaps we document this concern that you mention for users providing their own custom config file?